app/javascript/components/Request/Request.jsx

```diff
@@ -0,0 +1,343 @@
+import React, { useState }from 'react'
+import { Form, Col, Button, Container } from 'react-bootstrap';
+import { RequestValidation } from './validation';
+
+import { Header } from '../Header';
+import './Request.scss';
+
+const Request = () => {
```

Comment:
What is this request form for?

Reply:
The form is for making a delivery request.

Reply:
I thought it will be automated through a microservice